server/templates/nl_interface_data.html

```diff
@@ -0,0 +1,40 @@
+{#
+  Copyright 2023 Google LLC
+ 
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+ 
+       http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ #}
+ {%- extends BASE_HTML -%}
+ 
+ {% set is_hide_full_footer = true %}
+ {% set is_hide_sub_footer = true %}
+ {% set main_id = 'nl-interface' %}
+ {% set page_id = 'nl-interface-page' %}
+ {% set title = 'NL Explorer' %}
+ {% set subpage_title = 'NL Explorer' %}
+ {% set subpage_url = url_for('nl.page') %}
+ 
+ {% block head %}
+ <link rel="stylesheet" href={{url_for('static', filename='css/nl_interface.min.css', t=config['VERSION'])}} >
+ <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+ {% endblock %}
+ 
+ {% block content %}
+ <div id="metadata" class="d-none" data-website-hash="{{ website_hash }}" data-placeholder-query="{{ placeholder_query }}"></div>
+ <div id="dc-nl-interface"></div>
+ {% endblock %}
+ {% block footer %}
+ <script
+   src={{url_for('static', filename='nl_interface_data.js', t=config['VERSION'])}}></script>
+ <script src="https://maps.googleapis.com/maps/api/js?key={{maps_api_key}}&libraries=places&language={{locale}}" async
```

Comment:
no need for maps api key?

Reply:
maps api key is used for the place overview chart

Reply:
got it..also that seems not useful in the context of NL svg? We will not return a map chart?

Reply:
this may cause an error to be thrown if we don't have the map api & the client is trying to render the map

Reply:
Good for now. but later we can restrict this page not handling place overview query (won't be a legit use case).